packages/jsx-email/src/renderer/render.ts

```diff
@@ -20,8 +20,19 @@ export const renderPlainText = async (
   return htmlToText(result, {
     formatters: {
       raw: (elem, _walk, builder) => {
-        if (elem.children.length && elem.children[0].type === 'comment') {
-          builder.addInline(unescapeForRawComponent(elem.children[0].data!.trim()));
+        if (!elem.children.length) return;
+        const child = elem.children[0] as any;
+        if (child.type === 'comment') {
+          // Preserve exact content; do not trim
+          builder.addInline(unescapeForRawComponent(child.data));
+          return;
+        }
+
+        // Fallback for marker-wrapped content serialized as a text node
+        if (child.type === 'text' && typeof child.data === 'string') {
+          const re = new RegExp(`${RAW_WRAPPER_START}([\\s\\S]*?)${RAW_WRAPPER_END}`);
```

Comment:
since you are adding a regular expression, it should be an `export const` and should have test(s) written around it.